GLAA.Services/LicenceApplication/LicenceApplicationViewModelBuilder.cs

```diff
@@ -147,5 +156,12 @@ public IList<LicenceApplicationViewModel> BuildLicencesForUser(string userId)
 
             return mapper.Map<IList<LicenceApplicationViewModel>>(licences);
         }
+
+        private bool GetIsSubmitted(Licence licence)
+        {
+            var currentStatus = licence.CurrentStatusChange.Status;
+
+            return currentStatus.Id == constantService.ApplicationSubmittedOnlineStatusId;
```

Comment:
This feels a bit suspect - probably need to think about this in the future.  I.e. what if its submitted over the phone , what if it was previously submitted in the past, etc.

Reply:
Yeah that's a good spot. We may need a flag against the status for which constitute `submitted` or similar.

Reply:
I don't think this is a use case (unless the status points say otherwise). There isn't a way something can be unsubmitted (although there may be a need for changes post submission)